tests/unit/lib/kwlib_test.sh

```diff
@@ -965,4 +965,25 @@ function test_show_verbose()
   assert_equals_helper 'Expected value of command' "$LINENO" "$cmd" "$output"
 }
 
+function test_create_shared_memory_dir()
```

Comment:
Change this test name to something more meaningful, something like: `test_check_if_create_shared_memory_dir_creates_dir_in_TMP_DIR`.